app/models/concerns/dates_in_past_validator.rb

```diff
@@ -1,10 +1,18 @@
 class DatesInPastValidator < ActiveModel::Validator
   def validate(record)
     if record.instance_of?(Era) && record.start_date && record.end_date
-      record.errors.add(:start_date, 'cannot be in the future') if record.start_date > Date.today
-      record.errors.add(:end_date, 'cannot be in the future') if record.end_date > Date.today
+      era(record)
     elsif record.instance_of?(Event) && record.date
-      record.errors.add(:date, 'cannot be in the future') if record.date > Date.today
+      event(record)
     end
   end
+
+  def era(record)
+    record.errors.add(:start_date, 'cannot be in the future') if record.start_date > Time.zone.today
+    record.errors.add(:end_date, 'cannot be in the future') if record.end_date > Time.zone.today
+  end
+
+  def event(record)
+    record.errors.add(:date, 'cannot be in the future') if record.date > Time.zone.today
+  end
```

Comment:
This was so rubocop stopped giving us 7 errors.  Now it's just the 2 `get_` errors